tests/cpp/test_cpp_client.py

```diff
@@ -62,7 +62,7 @@ def execute_cmd(cmd_list):
     proc = Popen(
         cmd_list, stderr=PIPE, stdout=PIPE, stdin=PIPE, cwd=run_path)
     try:
-        out, err = proc.communicate(timeout=120)
+        out, err = proc.communicate(timeout=180)
```

Comment:
Any reason in particular why this was increased?

Reply:
Also, we should make sure we make these magic numbers more obvious.

Reply:
The CPP unit tests are timing out occasionally in the checkin, so I bumped it to 3 minutes from 2, Should really have done this in a separate PR but I was being lazy.  I agree in general about magic numbers, but this is the driver for our test cases, so not production code

Reply:
Agreed. Mainly just a note to myself to think about how to consolidate these kinds of things.